pxtarget.json

```diff
@@ -123,6 +123,44 @@
         "serviceId": "codal2cp"
     },
     "variants": {
+        "mkr1300": {
+            "serial": {
+                "useHF2": false
+            },
+            "compile": {
+                "isNative": true,
+                "useUF2": true,
+                "webUSB": false,
+                "hasHex": true,
+                "deployDrives": ".*",
+                "deployFileMarker": "INFO_UF2.TXT",
+                "driveName": "DRIVE",
+                "openocdScript": "source [find interface/cmsis-dap.cfg]; set CHIPNAME at91samd21g18; source [find target/at91samdXX.cfg]",
+                "flashChecksumAddr": 8372,
+                "flashCodeAlign": 256,
+                "upgrades": []
+            },            
+            "compileService": {
+                "buildEngine": "codal",
+                "codalTarget": {
+                    "name": "codal-mkr1300",
+                    "url": "https://github.com/ElectronicCats/codal-mkr1300",
```

Comment:
@pelikhan I do not know whether to place the original repository https://github.com/lancaster-university/codal-mkr1300 or the ElectronicCats repository, we as electroniccats can maintain our repository

Reply:
ok, for me it's okay

Reply:
seems to work for me.